ApiBundle/Transformer/TrashItemTransformer.php

```diff
@@ -37,6 +38,13 @@ public function transform($trashItem)
             ));
         }
 
+        if ($this->authorizationChecker->isGranted(AdministrationPanelStrategy::ROLE_ACCESS_DELETED_TRASHCAN)) {
+            $facade->addLink('_self_delete',  $this->generateRoute(
```

Comment:
Suggeston: when ``_self_delete`` is used on the platform, it means a soft delete. Maybe to mark the difference with the hard delete here, we could talk about ``_self_remove``